### DISCUSSION TITLE
NodeJs driver port from OrientDB

### DISCUSSION THREAD

**Discussion by qualitesys** [ArcadeData/arcadedb#402] [General]
As an OrientDB user, a would know if you have planned to port the nodejs driver called OrientJS to ArcadeDb ?
Thanks for your answer

https://orientdb.com/docs/last/orientjs/OrientJS.html

**Comment by lvca** [comment]
@qualitesys unfortunately no porting is scheduled right now. Some users are dealing with ArcadeDB server by using raw HTTP/JSON. ArcadeDB HTTP protocol is super simple and 99% of the things are managed by sending a command request with SQL/Gremlin/Cypher. Any volunteer to create a HTTP/JSON wrapper for ArcadeDB? :-)
